sidecar/src/figwheel_sidecar/build_middleware/javascript_reloading.clj

```diff
@@ -35,7 +71,7 @@
         (and (.exists out-file)
```

Comment:
I have just noticed there is a bug here, if the file does not exists the function returns a `boolean` (false) while the caller expects a `seq` of namespaces.